curated_transformers/generation/state.py

```diff
@@ -66,11 +110,13 @@ def _remove_completed(self, completed: Tensor):
             Tensor indicating for the active sequences whether they are completed.
         """
         not_completed = completed.logical_not()
+        self.generated_ids = self.generated_ids[not_completed]
         self.attention_mask = self.attention_mask[not_completed]
         if self.cache is not None:
             self.cache = [
                 layer_cache.filter_batch_items(not_completed)
                 for layer_cache in self.cache
             ]
+        self.prompt_ids = self.prompt_ids[not_completed]
```

Comment:
We don't really need to update `prompt_ids`, right? This is just to keep the shapes consistent, I take?